liquid.py

```diff
@@ -256,17 +293,49 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def initcollection(name):
+    """Initialize collection with given name.
+    
+    Create the snoop database, create the search index, run dispatcher, add collection
+    to search.
+    
+    :param name: the collection name
+    """
+    if name not in config.collections:
+        raise RuntimeError('Collection %s does not exist in the liquid.ini file.', name)
+
+    snoop_nomad_service_name = f'snoop-{name}-api'
+
+    try:
```

Comment:
Remove this `try/except/catch` because it's only printing the error to `stderr`. The python interpreter will do the same thing.

Reply:
Nope. The CalledProcessError exception stack trace does not contain the output from the command ran in the hoover search service. In case something went wrong I need to print that output, otherwise I won't know what happened.